lib/components/atoms/Track/types/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export type Props = {
+  src: String,
+  srcLang: String,
+  id: Number,
```

Comment:
Is it necessary to have Id as number and not as string?

Reply:
It can be both string and number, but should be unique
So updating id: number | string

Reply:
thanks, it will be number fixed